src/main/java/com/rollbar/android/Notifier.java

```diff
@@ -39,8 +39,8 @@
     private static final String ANDROID = "android";
     private static final String MESSAGE = "message";
 
-
-    private static final int DEFAULT_ITEM_SCHEDULE_DELAY = 1;
+    private static final int DEFAULT_ITEM_SCHEDULE_STARTUP_DELAY = 1;
```

Comment:
Can we just remove this? I don't see any benefit in waiting to send previous errors on app startup.